proxy/core/src/main/resources/dev-mode.conf

```diff
@@ -0,0 +1,5 @@
+# Configuration for using an in memory journal and snapshot store
+include classpath("in-memory.conf")
```

Comment:
@jroper I've removed the application.conf now and so you'll have to supply a -Dconfig.file|resource|url when launching the binary, for instance: -Dconfig.resource=dev-mode.conf